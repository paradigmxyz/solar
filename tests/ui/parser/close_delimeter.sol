struct X {
    uint y) //~ ERROR: `;`, found `)`
}