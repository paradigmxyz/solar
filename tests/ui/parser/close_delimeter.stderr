error: expected `;`, found `)`
  --> ROOT/tests/ui/parser/close_delimeter.sol:LL:CC
   |
LL |     uint y)
   |           ^ expected `;`

error: aborting due to 1 previous error

